BrightFuturesTests/ResultTests.swift

```diff
@@ -21,8 +21,18 @@
 // SOFTWARE.
 
 import XCTest
+import Result
 import BrightFutures
 
+extension Result {
+    var isSuccess: Bool {
```

Comment:
I think they help for readability in the tests (and it was less work this way ;) ), that's why I've added them here. After learning about analysis, I don't think they're needed in the library itself, though.